restcountries/base.py

```diff
@@ -7,20 +7,35 @@ class RestCountryApi:
     QUERY_SEPARATOR = ','
 
     @classmethod
-    def _get_country_list(cls, resource, term=''):
+    def _get_country_list(cls, resource, term='', filters=None):
         """Takes a resource and a search term and return a list of countries or a country.
 
         :param resource - resource to create the URL
         :param term - search term provided by the user of this package
+        :param filters - alist of fields to filter the output of the request to include only the specified fields.
```

Comment:
`alist` should probably be `a list`. This typo is present in every docstring.

Reply:
On it @SteinRobert  let me push the changes